imports/plugins/included/social/client/components/facebook.js

```diff
@@ -88,7 +88,9 @@ class FacebookSocialButton extends Component {
     });
 
     return (
-      <a className="btn btn-flat facebook-share" href="#" onClick={this.handleClick} target="_blank">
+      <a className="btn btn-flat facebook-share" aria-label="Share to Facebook" href="#" onClick={this.handleClick}
```

Comment:
Good call! I'll make a separate namespace for all of the aria translations as well. 👍